hw/ip/otbn/doc/_index.md

```diff
@@ -490,6 +490,35 @@ The error that caused the alert can be determined by reading the {{< regref "FAT
 If OTBN was running, this value will also be reflected in the {{< regref "ERR_BITS" >}} register.
 A fatal alert can only be cleared by resetting OTBN through the `rst_ni` line.
 
+
+### Error Prioritization
+
+OTBN can signal multiple errors at once, by setting multiple bits of {{< regref "ERR_BITS" >}}.
+There are some scenarios where one error takes priority over another.
+In these scenarios the higher priority error will suppress some other errors.
+
+* Any error that produces a fatal alert will suppress all non fatal errors.
+* Any `call_stack` error that occurs due to a pop will suppress:
+  - `bad_data_addr`
+  - `bad_insn_addr`
+  - `loop`
+* Any `illegal_insn` error will suppress:
+  - `bad_data_addr`
+  - `bad_insn_addr`
+  - `call_stack`
+  - `loop`
+
+(See {{< regref "ERR_BITS" >}} for the error meanings).
+
+Some combinations of error bits **cannot occur** in a correctly functioning OTBN implementation.
```

Comment:
Nit:
```suggestion
Some error combinations **cannot occur** in a correctly functioning OTBN implementation.
```

(See above; the "error bit" is just a representation of the actual error.)